daphne_worker/src/storage_proxy.rs

```diff
@@ -0,0 +1,136 @@
+use daphne_service_utils::do_requests::{DoRequest, IdFrom, DO_PATH_BASE, KV_PATH_BASE};
+use url::Url;
+use worker::{js_sys::Uint8Array, Env, Request, RequestInit, Response};
+
+const KV_BINDING_DAP_CONFIG: &str = "DAP_CONFIG";
+
+pub async fn handle_request(
+    req: Request,
+    env: Env,
+    _ctx: worker::Context,
+) -> worker::Result<Response> {
+    let path = req.path();
+    if let Some(uri) = path.strip_prefix(KV_PATH_BASE) {
```

Comment:
What are we using KV for? If DAP tasks (i.e., control plane) We discussed replacing it with D1. Where are we on that discussion?

Reply:
This service is simply a proxy for kv. We are using kv just as much in this implementation as we were. I don't think it makes sense to make any architecture changes in this rewrite to keep it as simple and testable as possible.